security/contextpolicy-config-handler/src/main/java/org/codice/ddf/admin/security/context/probe/AvailableOptionsProbeMethod.java

```diff
@@ -106,16 +114,16 @@ public ProbeReport probe(ContextPolicyConfiguration config) {
     }
 
     public List<String> getRealms() {
-        List<String> realms = new ArrayList<>(Arrays.asList(KARAF));
+        List<String> realms = new ArrayList<>(Collections.singletonList(KARAF));
```

Comment:
There's a number of other places we should probably make this change in Sources as well.